Makefile

```diff
@@ -25,7 +25,7 @@ AAR_FILE_DIR =  $(ROOT_DIR)/code/$(PROJECT_NAME)/build/outputs/aar
 
 init:
 	git config core.hooksPath .githooks
-	
+
 clean:
```

Comment:
@praveek and @yangyansong-adbe Can you guys help with reviewing changes for dokka. I added a first Kotlin file in the source for Edge and have made changes for that in
`Makefile`, `build.gradle` for the project (edge-sdk), `build.gradle` for edge module (edge). Did not update the Github workflow.

Reply:
@addb  the dokka setting looks good to me. Before merging the PR, please run the `javadocJar` command locally and check the generated HTML files, in case there are some bad links or missing API documentation.

Reply:
Thanks @yangyansong-adbe